python/pyproject.toml

```diff
@@ -23,7 +23,7 @@ dependencies = [
 srt = ["aiohttp", "decord", "fastapi", "hf_transfer", "huggingface_hub", "interegular",
        "packaging", "pillow", "psutil", "pydantic", "python-multipart",
        "torch", "torchao", "uvicorn", "uvloop", "zmq",
-       "vllm==0.5.5", "outlines>=0.0.44", "modelscope"]
+       "vllm==0.6.3", "outlines>=0.0.44", "modelscope"]
```

Comment:
I see that you have updated the version of vLLM here. Is there anything blocking this PR at the moment?

Reply:
there are some modification for VllmModelConfig in latest version to adapt qwen2-vl, if not updating vllm, it will cause error when loading qwen2-vl, but it seems VllmModelConfig is not easy to move from vllm
https://github.com/sgl-project/sglang/blob/56503d9bc93a13db22b9314a93c83a4f2ce4c362/python/sglang/srt/model_executor/model_runner.py#L233-L242